lib/server.js

```diff
@@ -794,6 +796,10 @@ WalletService.prototype.createTx = function(opts, cb) {
   if (!Utils.checkRequired(opts, ['toAddress', 'amount', 'proposalSignature']))
     return cb(new ClientError('Required argument missing'));
 
+  var feePerKb = opts.feePerKb || 10000;
+  if (feePerKb > 10000)
```

Comment:
I didn't want to prematurely rule out free transactions as there are some conditions under which this is acceptable (see https://en.bitcoin.it/wiki/Transaction_fees#Sending). But since we are not checking the other conditions I think your suggestion provides more safety. Will change.